perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -775,30 +772,37 @@ sub construct_bin_request_form {
 
 sub request : Chained('property') : Args(0) {
     my ($self, $c) = @_;
-    my $field_list = construct_bin_request_form($c);
 
-    $c->stash->{first_page} = 'request';
-    my $next = $c->cobrand->call_hook('waste_request_form_first_next');
-    my $title = $c->cobrand->call_hook('waste_request_form_first_title') || 'Which containers do you need?';
+    my %form_settings
+        = $c->cobrand->call_hook( 'construct_bin_request_form', $c );
+
+    my $field_list
```

Comment:
I think this four-line line is more easily parsable as `my $field_list = $form_settings{field_list} || construct_bin_request_form($c);` :)